yarn.lock

```diff
@@ -2110,6 +2110,17 @@
     react-syntax-highlighter "^12.2.1"
     regenerator-runtime "^0.13.3"
 
+"@storybook/addon-toolbars@^6.0.27":
```

Comment:
I added this in another PR but accidentally removed it in the #454 bump. This adds it back.